blog.html

```diff
@@ -0,0 +1,251 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Our Blog - Binabox</title>
+    <link rel="stylesheet" href="normalize.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <a href="#" class="logo">
+            <img src="images/logo-white.svg" width="188" height="42" class="logo__image" alt="Binabox">
+        </a>
+        <nav class="header-nav">
+            <ul class="header-nav__wrapper">
+                <li class="header-nav__item">
+                    <a href="#">Home</a>
+                </li>
+                <li class="header-nav__item">
+                    <a href="#">Blog</a>
+                </li>
+                <li class="header-nav__item">
+                    <a href="#">Catalog</a>
+                </li>
+                <li class="header-nav__item">
+                    <a href="#">Contact Us</a>
+                </li>
+            </ul>
+        </nav>
+        <div class="header-actions">
+            <form class="search-form" action="/search">
+                <button type="submit" class="search-form__btn">
+                    <img src="images/search.svg" width="24" height="24" alt="search">
+                </button>
+                <input class="search-form__input" type="text" placeholder="Search" aria-label="Search"> 
+            </form >
+            <a href="#" class="btn btn--icon">
+                <img src="images/discord.svg" width="24" height="24" alt="discord">
+                Discord
+            </a>
+            <a href="#" class="btn btn--icon btn--color">
+                <img src="images/connect.svg" width="24" height="24" alt="connect">
+                Connect
+            </a>
+        </div>
+    </header>
+    <section class="blog-intro">
+        <h2 class="blog-intro__title">Our blog</h2>
+        <ul class="blog-intro__nav">
+            <li href="#" class="blog-intro__item">Home</li>
+            <li href="#" class="blog-intro__item">Our blog</li>
+            <li href="#" class="blog-intro__item">The Basilisks NFT</li>
+        </ul>
+    </section>
+     <main class="blog-wrapper">
```

Comment:
class="main main--two-col"
It's can be useful not for blog page only

Reply:
done